assets/methods_description_template.yml

```diff
@@ -1,4 +1,4 @@
-id: "genomic-medicine-sweden-metaval-methods-description"
+id: "genomic-medicine-sweden-meta-val-methods-description"
```

Comment:
here you have `meta-val`

Reply:
Well, I'm going to open a PR to correct the name of my pipeline. This is really annoying to me